guardian/s3-upload
# Add node to Snyk workflow

## What does this change?

Adds node to Snyk workflow – there's a project in public/js that should be scanned.

## How to test

Run the workflow manually in GHA. You should see the [project appear.](https://app.snyk.io/org/guardian/project/b7c03694-a35d-4abf-ad7f-8db0c0b3f380/history/fac2ad51-26d4-44cd-9c13-c26a2e4da167)
